packages/web/src/components/collectibles/components/CollectibleDetails.tsx

```diff
@@ -120,7 +120,6 @@ const CollectibleDetails = (props: CollectibleDetailsProps) => {
                 (mediaType === CollectibleMediaType.VIDEO && frame)) && (
                 <div className={styles.imageWrapper}>
                   <PreloadImage
-                    asBackground
```

Comment:
this is so that the <img> tag is used in the component, instead of a div with background url